### PR TITLE
stop using 'remove_usr_bin'-patch in TensorFlow easyconfig, no longer required with updated TensorFlow easyblock

### PR DESCRIPTION
**Pull request by Flamefire**
Handled by the EasyBlock now in a more general way

Follow-up to ~~https://github.com/easybuilders/easybuild-easyblocks/pull/2110~~

Note: For 1.12.0 the patch is still there but I considered them too old (2018b toolchain)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -22,6 +22,10 @@ dependencies = [
     ('h5py', '2.9.0'),
 ]
 
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
```
Comment (boegel) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2019a-Python-3.7.2.eb:26]: @Flamefire Please also clean up the `source_urls` in `exts_list`, at least if they point to PyPI...

Comment (Flamefire) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2019a-Python-3.7.2.eb:26]: Done

**Comment by Flamefire** [comment]
Readded the patch and adjusted the commit message

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 16 out of 16 (16 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/4557dcaed20ffc3f4b1604646866efed for a full test report.

**Comment by boegel** [comment]
That test report was only with `eb --fetch`, to make sure auto-downloading didn't get broken.

Full test report with actual installations coming up (but it'll take a while...)

**Comment by boegel** [comment]
@smoors I've cancelled the job submitted by @boegelbot, there's too much dependencies missing, it'll take forever.

I'll install the missing deps centrally first on `generoso`, and then let the bot re-try...

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 16 out of 16 (16 easyconfigs in this PR)
node3303.joltik.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/43efd4999109d0388bf6bcd7b5282154 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 14 out of 16 (16 easyconfigs in this PR)
node3166.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/457413cc8b497b837a50b4f402048aa5 for a full test report.

**Comment by boegel** [comment]
Problems in last test report with the TensorFlow 1.14.0 easyconfigs have been fixed in b3151d9 (the test started running before that commit was added), I'll submit a new test report for just those two easyconfigs to confirm...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/d5bc9fcc75aa243f7da00f3a3b576006 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Flamefire!

**Comment by Flamefire** [comment]
@boegel although your bot thinks otherwise I'd say it has a bug ;D

**Comment by boegel** [comment]
> @boegel although your bot thinks otherwise I'd say it has a bug ;D

I agree, looking into it... Bad bot!

**Pull request merged**